patch/README.md

```diff
@@ -13,4 +13,15 @@ The `*-ngx_pipe_environ_on_mac` patches support the environ argument of the ngx.
 
 The `*-enable_keepalive` patches originally come from:
 https://github.com/openresty/lua-nginx-module/pull/1600
-https://github.com/openresty/lua-resty-core/pull/276
\ No newline at end of file
+https://github.com/openresty/lua-resty-core/pull/276
+
+The `*-ngx_meta_lua_module` patches, `src/meta/*` and `t/meta/*` originally come from:
+https://github.com/openresty/meta-lua-nginx-module/pull/76, which is under BSD license.
```

Comment:
We should mention other PRs, like the one to lua-resty-core.

Reply:
add commit link